core/src/setup/Kokkos_Setup_SYCL.hpp

```diff
@@ -17,6 +17,12 @@
 #ifndef KOKKOS_SETUP_SYCL_HPP_
 #define KOKKOS_SETUP_SYCL_HPP_
 
+// FIXME SYCL+Cuda Apparently, submit_barrier doesn't quite work as expected
```

Comment:
FIXME is fine but me but just want to make sure you did not want FIXME_SYCL

Reply:
Changed it to FIXME_SYCL, see da972211c1152f69cb2b38d4654db46888417590.